data/migrations/20211111164703_mentor_intake.js

```diff
@@ -13,10 +14,11 @@ exports.up = function (knex) {
         .onUpdate('RESTRICT');
       table.string('email').notNullable();
       table.string('location').notNullable();
-      table.string('name').notNullable();
+      table.string('first_name').notNullable();
+      table.string('last_name').notNullable();
       table.string('current_comp');
       table.string('tech_stack').notNullable();
-      table.boolean('can_commit').notNullable();
+      table.string('can_commit').notNullable();
```

Comment:
Why was this changed? I think the `can_commit` value _should_ be true/false, as the value that follows it seems like where people would explain _how_ they can commit to provide additional context.

Reply:
I think youre right, it would make more sense to be true/false. While testing the interaction with this table from the frontend though, we found that currently the 'can_commit' question accepts three radio inputs which are taken in as strings. Only when 'neither' is selected is the 'how_commit' input field revealed. So by changing this to a string the user input, as it is now, can be passed to our db.